File: diKoding/DotPeekServer/DotPeekServerCs/Blog.cs
namespace DotPeekServer
{
    using System;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    [Table(@"Blogs")]
    public sealed class Blog
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Int32 Id { get; set; }

        [StringLength(150)]
        public String Title { get; set; }

        public DateTime PostAt { get; set; }

        [StringLength(50)]
        public String PostBy { get; set; }

        public override String ToString()
        {
            return String.Format(
                @"{0}-{1}-{2}-{3}",
                this.Id,
                this.Title,
                this.PostAt,
                this.PostBy);            
        }
    }
}

File: diKoding/DotPeekServer/DotPeekServerVb/Blog.vb
Imports System
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

<Table("Blogs")> _
Public NotInheritable Class Blog
    <DatabaseGenerated(DatabaseGeneratedOption.Identity)> _
    Public Property Id As Int32

    <StringLength(150)> _
    Public Property Title As [String]

    Public Property PostAt As DateTime

    <StringLength(50)> _
    Public Property PostBy As [String]

    Public Overrides Function ToString() As [String]
        Return [String].Format( _
            "{0}-{1}-{2}-{3}", _
            Id, _
            Title, _
            PostAt, _
            PostBy)
    End Function
End Class